daml-lf/validation/src/test/scala/com/digitalasset/daml/lf/validation/TypingSpec.scala

```diff
@@ -65,9 +65,6 @@ class TypingSpec extends AnyWordSpec with TableDrivenPropertyChecks with Matcher
         BTRoundingMode -> k"*",
```

Comment:
Can we add some tests for the new primitives?

Reply:
Added!